synfig-studio/src/gui/docks/dock_info.cpp

```diff
@@ -207,6 +232,11 @@ void studio::Dock_Info::changed_canvas_view_vfunc(etl::loose_handle<CanvasView>
 	}
 }
 
+void studio::Dock_Info::set_async_render(etl::handle<studio::AsyncRenderer> ar)
+{
+	async_renderer = ar;
```

Comment:
As of now the button is enabled only as there is something that is getting rendered:
https://github.com/synfig/synfig/blob/086ae7d1a854220a829915b08166b25a12333618/synfig-studio/src/gui/docks/dock_info.cpp#L254-L257

Even if someone calls this with a null pointer, it would still be not sensitive anyway.
Did I miss something or you agree?

Reply:
Code changes…
For now, only one place calls the Dock_Info::set_async_renderer(), but who knows?
Better let it "future-proof" :) Simple to solve.
`if (!ar)` for example.

Reply:
Right!

Reply:
Anyway thank you @rodolforg !
I'm learning a lot of interesting stuff thanks to your tips :+1:

Reply:
I'm glad to help you :)